sdcm/nemesis.py

```diff
@@ -4043,19 +4044,24 @@ def _decommission(node):
         InfoEvent(f'FinishEvent - ShrinkCluster has done decommissioning {len(nodes)} nodes').publish()
 
     def _decommission_nodes(self, nodes_number, rack, is_seed: Optional[Union[bool, DefaultValue]] = DefaultValue,
-                            dc_idx: Optional[int] = None):
+                            dc_idx: Optional[int] = None, exact_nodes: list[BaseNode] | None = None):
```

Comment:
we do gonna put both parameter at the same time, one is overriding the other
I don't think copying it would be more helpful in this case

so far we didn't had any requirement for this nemesis to no decommission the same amount of nodes